bin/utils/zowe-variable-utils.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+
+################################################################################
+# This program and the accompanying materials are made available under the terms of the
+# Eclipse Public License v2.0 which accompanies this distribution, and is available at
+# https://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Copyright IBM Corporation 2020
+################################################################################
+
+#TODO LATER - provide flag that toggles all functions to error if they exit non-zero?
+
+# Takes in two parameters - the name of the variable (for error messaging) and the value
+validate_variable_is_set() {
+  variable_name=$1
+  value=$2
```

Comment:
I think here we better use `eval "value=\"\$${variable_name}\""` to get the value of `variable_name`. Otherwise this check relies on if the caller can provide a valid `$2`.

Reply:
That's way better - I'll respin with that change in